kernel_gateway/gatewayapp.py

```diff
@@ -494,12 +505,12 @@ def _build_ssl_options(self):
             ssl_options['keyfile'] = self.keyfile
         if self.client_ca:
             ssl_options['ca_certs'] = self.client_ca
+        if self.ssl_version:
+            ssl_options['ssl_version'] = self.ssl_version
         if not ssl_options:
             # None indicates no SSL config
             ssl_options = None
         else:
-            # SSL may be missing, so only import it if it's to be used
-            import ssl
             # Disable SSLv3 by default, since its use is discouraged.
             ssl_options.setdefault('ssl_version', ssl.PROTOCOL_TLSv1)
```

Comment:
You could then update this to reference `self.ssl_version_default_value`.  We should probably delete the `# Disable` comment as well.

I'm not sure how critical it was to defer the import of SSL, but it _could_ just be referenced here and forgo its setup on `self.ssl_version_default_value` in that case.  But it appears to be part of the standard Python library, so it doesn't seem like a big deal to defer its import.  (I'm still relative new to Python, so I might be missing something about this.)